canonical/ubuntu.com
# Switched from ip to tz for geolocation

## Done

- Switched from using ip address for geolocation to using timezone

## QA

- Open https://ubuntu-com-13636.demos.haus/contact-us/form?product=generic-contact-us 
  - Check that the phone number input shows the correct country flag for your location.  
  - Be sure to test on mobile, tablet and desktop screen sizes
  
## Issue / Card

Fixes [WD-1173](https://warthogs.atlassian.net/browse/WD-1173)

## Screenshots

![image](https://github.com/canonical/ubuntu.com/assets/27333173/ff72cf85-d738-4e79-b7b2-6cbd9692a3aa)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

## Follow up
Should we also auto-fill the country input? 


[WD-1173]: https://warthogs.atlassian.net/browse/WD-1173?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
